packages/core/lib/debug/printer.js

```diff
@@ -209,11 +210,43 @@ class DebugPrinter {
       this.config.logger.log(DebugUtils.formatStack(step.stack));
       this.config.logger.log("");
     }
-    this.config.logger.log(
-      DebugUtils.formatInstruction(traceIndex + 1, totalSteps, instruction)
-    );
-    this.config.logger.log(DebugUtils.formatPC(step.pc));
+
+    this.config.logger.log("Instructions:");
+    this.config.logger.log("");
```

Comment:
Ah, sorry, one last thing before merging -- could you get rid of this extra blank line?  I don't think we need this, I think we want it more compact.  Thanks!

Reply:
changes made